### PR TITLE
#3: Proposal to make compatible with sass `@media`. Other refactors and improvements.

### PR DESCRIPTION
**Pull request by blainehansen**
extend placeholders now call mixins rather than the other way around; sass is used instead of scss; space-evenly was added to justify-content and align-content; general refactoring and cleaning for DRY code; bumped to 2.0.0 (since there are major api changes), and added blaine hansen to contributors list; changed readme to reflect; added dev directory and dependencies to allow in-package testing; added `.editorconfig` to make github viewing more readable.

### CODE REVIEW

**Comment by zessx** [comment]
Thanks for your PR, and sorry to not have taken the time to update it myself, will review it today.   
I'm not really ok with the SCSS syntax drop (as this is the mainly used one) but we can easily make both versions cohabit.

File: README.md [hunk 1]
```diff
@@ -5,113 +5,186 @@
...
+# or 
+
+bower install zessx-sass-flexbox
+```
+
+```sass
+@import 'whatever/path/to/modules/sass-flexbox'
+```
 
-    @import "helpers/sass-flexbox";
```
Comment (zessx) [inline README.md:27]: Add a block to explain the test procedure:

> ## Testing
> 
> You can test SassFlexbox, using NPM scripts:
> 
> ```
> npm install --only=dev 
> npm run-script output-dev
> # >> dev/output.css
> ```

File: sass-flexbox.sass [hunk 2]
```diff
@@ -0,0 +1,309 @@
+$sass-flexbox-rename-map: (nowrap: none, auto: auto, flex-start: start, flex-end: end, center: center, baseline: baseline, space-between: space-between, space-around: space-around, stretch: stretch, inherit: inherit)
```
Comment (zessx) [inline sass-flexbox.sass]: Leave both `.sass`and `.scss` files in the repository.

File: README.md [hunk 3]
```diff
@@ -5,113 +5,186 @@
...
+```bash
+npm install sass-flexbox --save 
+
+# or 
+
+bower install zessx-sass-flexbox
+```
+
+```sass
+@import 'whatever/path/to/modules/sass-flexbox'
```
Comment (zessx) [inline README.md:24]: If we leave both files, the file extension will be required, and it needs to be specified in the documentation:

> Include the file in your project, specifying the extension (either `sass` or `scss`):
> 
> ```sass
> @import 'whatever/path/to/modules/sass-flexbox.sass'
> ```

**Review by zessx — CHANGES_REQUESTED** [review]
Nothing to say on the code itself, I successfully tested it, thanks!

**Comment by blainehansen** [comment]
How is this looking now? :)

**Review by zessx — APPROVED** [review]

**Comment by zessx** [comment]
Fix #3

**Pull request merged**